impls/src/adapters/emoji.rs

```diff
@@ -13,1041 +13,174 @@
 // limitations under the License.
 
```

Comment:
In here too, add file comments explaining the scope of the file using `//!´

Reply:
Of course